refactor: replace incoming/outgoing transaction icons

dated opened the pull request:
<!--
Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!

Please make sure you're familiar with and follow the instructions in the [contributing guidelines](https://ark.dev/docs/program-incentives/guidelines/contributing).

Please fill out the information below to expedite the review and (hopefully) merge of your pull request!
-->

## Summary

https://app.clickup.com/t/mdcgkv

<!-- What changes are being made? -->

<!-- Why are these changes necessary? -->

<!-- How were these changes implemented? -->

## Checklist

<!-- Have you done all of these things?  -->

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

<!-- Feel free to add additional comments. -->

dated committed:
refactor: replace incoming/outgoing transaction icons

faustbrian commented:
Task linked: [CU-mdcgkv [Icon Update] New Incoming/Outgoing icons in the table](https://app.clickup.com/t/mdcgkv)

boldninja committed:
chore: optimize 2 SVG(s)

Optimized SVG(s):
- src/app/assets/svg/received.svg
- src/app/assets/svg/sent.svg

faustbrian committed:
Merge branch 'develop' into refactor/tx-icons-CU-mdcgkv
